tst/EnergyPlus/unit/SQLite.unit.cc

```diff
@@ -590,35 +590,35 @@ TEST_F(SQLiteFixture, SQLiteProcedures_createZoneExtendedOutput)
     zoneData1->MaximumZ = 2;
     zoneData1->CeilingHeight = 2;
     zoneData1->Volume = 2;
-    zoneData1->InsideConvectionAlgo = 2;
-    zoneData1->OutsideConvectionAlgo = 2;
+    zoneData1->IntConvAlgo = Convect::HcInt::ASHRAETARP;
+    zoneData1->ExtConvAlgo = Convect::HcExt::ASHRAETARP;
     zoneData1->FloorArea = 2;
     zoneData1->ExtGrossWallArea = 2;
     zoneData1->ExtNetWallArea = 2;
     zoneData1->ExtWindowArea = 2;
     zoneData1->isPartOfTotalArea = false;
 
-    auto const &zoneListData0 = std::make_unique<DataHeatBalance::ZoneListData>();
+    auto const zoneListData0 = std::make_unique<DataHeatBalance::ZoneListData>();
```

Comment:
We need to have a talk about `auto &`. And a separate talk about unique pointers.